### ISSUE TITLE
Pandoc-compability of Latex export - tables

### ISSUE THREAD

**Issue by nmueller18** [fiduswriter/fiduswriter#1235]
Currently, the latex exporter uses `tabu` for tables. This works for proper Latex-compilers but not for Pandoc which seems to rely on `tabular`.
The following codes with `tabu` does not generate proper tables (with e. g. `pandoc -s -f latex -t docx -o document.docx document.tex`):

`\begin{document}
\begin{table}[h!]
\begin{tabu}{ |X|X| }
x&y\\
z&a\\
\end{tabu}
\caption{A caption}
\end{table}
\end{document}`

The following with `tabular` does:
`\begin{document}
\begin{table}[h!]
\begin{tabular}{ll}
x&y\\
z&a\\
\end{tabular}
\caption{A caption}
\end{table}
\end{document}`

Is there a deeper reason to use `tabu` instead of `tabular` for tables in the Latex exporter?

**Comment by johanneswilm** [comment]
Hey @nmueller18 ,
that is a good question. In this commit https://github.com/fiduswriter/fiduswriter/commit/f25f772ace5d67cac9e70727915c28ee7fc4705e we switched from `tabularx` to `tabu`. As far as I can tell, that is because with `tabu` we could get right/left alignment to work correctly. I cannot find that we ever tried to use `tabular`. Based on what I can see in this wiki [1], it sounds like the basic `tabular` package doesn't have much of the functionality of the more advanced options. However, it's possible that it would just work.

[1] https://en.wikibooks.org/wiki/LaTeX/Tables#Width_and_stretching

**Comment by nmueller18** [comment]
Yes, it seems that `tabular` is very basic. For our workflow from Latex to Indesign via Pandoc, it is sufficient, though.

**Comment by johanneswilm** [comment]
I see. That is probably we moved away from it. Maybe it would be possible to use `tabular` in those cases where none of the extra features of `tabu` are in use. The long-term solution is probably still direct pandoc export.

**Comment by nmueller18** [comment]
It seems that `tabu` is in maintenance mode and a [dead end](https://github.com/tabu-issues-for-future-maintainer/tabu) ...

**Comment by johanneswilm** [comment]
I was looking at this in early 2023 when last looking at this ticket. As far as I remember, even though it is in maintenance mode, there is no good alternative yet that covers all of its features. Do you have other information @nmueller18 ? Should I look for alternatives again?

**Comment by johanneswilm** [comment]
@nmueller18 Any experiences with https://github.com/lvjr/tabularray ? It seems to be one of the best maintained table packages currently.

**Comment by nmueller18** [comment]
Sorry, no, I have no experience `tabularray` and also no other recommendations.

**Comment by johanneswilm** [comment]
Not to be fixed now that we have direct pandoc export
